src/controller/java/src/chip/devicecontroller/ControllerParams.java

```diff
@@ -159,6 +169,11 @@ public Builder setIpk(byte[] ipk) {
       return this;
     }
 
+    public Builder setAdminSubject(long adminSubject) {
```

Comment:
Documentation for this setter? What are the effects of setting this, or leaving it at its defaults?

Reply:
Added comments